conda-env-dev.yml

```diff
@@ -0,0 +1,10 @@
+name: primo
```

Comment:
Just curious. Is there a way to pass the environment name? `primo` can be the default. But if I need to create a new environment to run some tests, it would be helpful if I can pass the environment name.